src/integration_free_fleet_adapter.md

```diff
@@ -0,0 +1,153 @@
+
+## ROS 2 Navigation2
+
+For ROS 2 Navigation2, the integration is enabled by [zenoh-bridge-ros2dds](https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds). With an appropriately configured bridge running on the robot, the `free_fleet_adapter` is able to communicate and control the robot, via its navigation behavior trees.
+
+![](https://github.com/open-rmf/free_fleet/blob/media/multirobot_sim_architecture.jpg?raw=true)
+
+![](https://github.com/open-rmf/free_fleet/blob/media/ff_unique_faster_smaller.gif?raw=true)
+
+The Navigation2 integration currently uses the simple `navigate_to_pose` behavior to provide navigation commands from point to point following the RMF waypoints defined. Support for utilizing more Navigation2 behaviors or custom behaviors is on the roadmap, but in the meantime, contributions are always welcome!
+
+Below is an example of the robot defined in the fleet configuration file, you can find the file [here](https://github.com/open-rmf/free_fleet/blob/main/free_fleet_examples/config/fleet/nav2_tb3_simulation_fleet_config.yaml). Take note of the defined `navigation_stack`, this allows the `free_fleet_adapter` to pick the `Nav2RobotAdapter` for this particular robot.
+
+```yaml
+...
+  robots:
+    nav2_tb3:
+      charger: "tb3_charger"
+      responsive_wait: False # Should responsive wait be on/off for this specific robot? Overrides the fleet-wide setting.
+      # For Nav2RobotAdapter
+      navigation_stack: 2
+      initial_map: "L1"
+      maps:
+        L1:
+          map_url: "/opt/ros/jazzy/share/nav2_bringup/maps/tb3_sandbox.yaml"
+,,,
+```
+
+Below is an example of the Zenoh configuration file used by the `zenoh-bridge-ros2dds` on the robot. The Zenoh keys are namespaced by the `namespace` parameter, while we are able to filter and rate-limit ROS 2 topics, services, actions to reduce network traffic. With the `client` mode, the bridge will seek to connect to a locally available Zenoh router.
+
+```json5
+{
+  plugins: {
+    ros2dds: {
+      namespace: "/nav2_tb3",
+      allow: {
+        publishers: [".*/tf", ".*/battery_state"],
+        subscribers: [],
+        service_servers: [],
+        service_clients: [],
+        action_servers: [".*/navigate_to_pose"],
+        action_clients: [],
+      },
+      pub_max_frequencies: [".*/navigate_to_pose/*=1", ".*/battery_state=1"],
+    },
+  },
+  // Zenoh related configurations
+  mode: "client",
+  connect: {
+    endpoints: []
+  },
+  listen: {
+    endpoints: []
+  },
+}
+```
+
+Check out our [integration test setup](https://github.com/open-rmf/free_fleet/blob/main/.github/docker/integration-tests/nav2-docker-compose.yaml) to get a better idea on how this integration can be deployed.
+
+## ROS 1 Navigation Stack
+
+<div class='warning'>
+The `zenoh-bridge-ros1` is an experimental bridge, and is actively being iterated on. Especially with <a href="https://wiki.ros.org/Distributions">ROS 1 going EOL in May 2025</a>, users are encouraged to migrate to ROS 2.
```

Comment:
The ticks around `zenoh-bridge-ros1` don't render correctly in the final mdbook when used inside the div. Not a big deal, but I thought I'd mention it. The name isn't ticked elsewhere so we could just remove the ticks here.

Reply:
Good point, thanks for catching that. Removed ticks in https://github.com/osrf/ros2multirobotbook/pull/114/commits/9b41750f8bf142e2c620f9d737f974493313b92b